core/docker/cli-arm64.dockerfile

```diff
@@ -0,0 +1,29 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+FROM centos:centos8
+
+RUN \
+    set -xeu && \
+    yum -y -q install java-11-openjdk-headless java-11-openjdk-devel && \
```

Comment:
We shouldn't need `-devel` here since this just the CLI, no need for tools

Reply:
Done! It changed the uncompressed image size by 6MB but made almost no change to the compressed image. Anyway, when we switch to JDK 17, we can use Zulu for both amd64 and arm64.